cinema/views.py

```diff
@@ -21,22 +22,33 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    mixins.CreateModelMixin, mixins.ListModelMixin, viewsets.GenericViewSet
```

Comment:
Place all parents on a new line to match the code style and improve the readability as it is done for `MovieViewSet` class. Apply this change to all edited classes